package.json

```diff
@@ -21,7 +21,7 @@
     "copy-files": "cp -R ./src/templates ./lib && cp -R src/codemods ./lib",
     "lint": "eslint . --cache --ext .ts",
     "prepare": "yarn build",
-    "test": "jest --runInBand"
+    "test": "jest --runInBand --silent"
```

Comment:
What does `--silent` do for us?

Reply:
It's _supposed_ to suppress the console output in tests, but it doesn't appear to do so (perhaps I misinterpreted). I can remove.